### ISSUE TITLE
Problem of BatchNormalization

### ISSUE THREAD

**Issue by Melcfrn** [eth-sri/eran#114]
Hi, 

I would like to congratulate your team for that nice job.

I used Deepzono on cifar10 dataset and with my own model. The model is trained with PyTorch and saved to ONNX format.
I discovered an error with BatchNorm2d layer. When there is that layer in the model, the accuracy in ERAN doesn't correspond to the accuracy in PyTorch. (When there is no BatchNorm2d, accuracies are equals). 
I saw in another issue that the code for BatchNormalization was proposed for DeepPoly so is it normal that it doesn't work for DeepZono ? (https://github.com/eth-sri/eran/issues/40#issue-739889155)

Best regards,
Clément
